This file is part of MXE.
See index.html for further information.

Contains a fix for issue 35 on json-c (https://github.com/json-c/json-c/issues/35)

--- a/Makefile.am	2012-07-17 22:52:19.000000000 +0200
+++ b/Makefile.am	2012-07-17 22:54:09.000000000 +0200
@@ -17,6 +17,7 @@
 	json_config.h \
 	json_inttypes.h \
 	json_object.h \
+	json_object_iterator.h \
 	json_object_private.h \
 	json_tokener.h \
 	json_util.h \
